generators/client/templates/vue/package.json.ejs

```diff
@@ -70,6 +70,9 @@
 <%_ } _%>
     "mini-css-extract-plugin": "^0.4.4",
     "node-notifier": "^5.1.2",
+    <%_ if (useSass) { _%>
```

Comment:
we could get rid of the conditions since it will always be true

Reply:
Yes, sure we can